front/app/components/HomeProjectList.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import ProjectDetail from './ProjectDetail'
+
+class HomeProjectList extends React.Component {
+
+    constructor() {
+
+        super();
+        this.render = this.render.bind(this);
+        this.state = {projects: []};
+
+    }
+
+    componentDidMount() {
+
+        var xhr = new XMLHttpRequest();
+        xhr.open('get', 'http://localhost:3000/projects', true);
+        xhr.onload = () => {
+            var data = JSON.parse(xhr.responseText);
+            this.setState(data);
+        };
+        xhr.send();
+    }
+
+    render() {
+
+        if (this.state.projects.length == 0) {
+            return <div>Loading projects...</div>
+        }
+
+        let projectComponents = [];
+
+        this.state.projects.forEach(project => {
```

Comment:
idem que pour les cardBlock : plutot que de passer par une variable mutable projectComponents et un forEach, passe directement par les méthodes fonctionnel de javascript : map => 
return this.state.projects.map(project => (<ProjectDetail project={project} />)

petit tips : j'ai mis un return parce que globalement quand tu fait ce genre de manipulation c'est cool de les sortir dans une fonction a part (au même niveau que rendre ou componentDidMount) pour améliorer la lisibilité. Par exemple renderProjects